integrations/airflow/documentation.yaml

```diff
@@ -40,7 +40,7 @@ podmonitoring_config: |
         component: statsd
         release: airflow
 additional_podmonitoring_info: |
-  Ensure that the values of the port and matchLabels match that of the {{app_name_short}} pods you want to monitor.
+  Ensure that the values of the port and matchLabels fields match those of the {{app_name_short}} pods you want to monitor.
```

Comment:
Sorry, I've been trying to suggest that "port" and "matchLabels" be formatted as literals, but the damn "comment" block takes out my tick-marks and format them as literals, and I think that makes it harder to see what the comment was. I hate "helpful" autoformatting; lemme try escaping them:

\`port\` and \`matchLabels\` fields

Reply:
OK, looks like that worked. Third time's a charm. So sorry!!

Reply:
Sorry, I'm still not seeing a change: \`port\` and \`matchLabels\` fields

(Could be my lack of Github foo, but I can't find the change.)

Reply:
@mtabasko not sure why the pull request is not showing the change. The change was already implemented on my end. I can see it here https://github.com/observIQ/monitoring-dashboard-samples/blob/gmp/doc/airflow/integrations/airflow/documentation.yaml#L43

Reply:
This is what I see:

Ensure that the values of the port and matchLabels fields match those of the {{app_name_short}} pods you want to monitor.

not

Ensure that the values of the \`port\` and \`matchLabels`\  fields match those of the {{app_name_short}} pods you want to monitor.

Is this some autoformatting issue, I wonder? (I've never noticed this before -- the tickmarks usually show up just fine in the YAML files (it's the comments where they are problematic).

Reply:
ahh, I understand. Pushed a change :facepalm: